includes/gateway/class-omise-payment-atome.php

```diff
@@ -202,7 +202,7 @@ private function getItems($order, $currency)
             $pricePerItem = $item['subtotal'] / $item['qty'];
 
             // Remove product from the list if the price is 0
-            if ((float)$pricePerItem === 0.00) {
+            if ((float)$item['subtotal'] === 0.00) {
```

Comment:
Why do we need to cast to `float`?

Reply:
So that the amount between 0 and 1 would be represented correctly. The minimum price can be satisfied by the shipping cost. This might not happen in real life but just in case.

Reply:
Since you are casting as float.
If amount is RM 0.8 then casting to float will not give 0, it will be still 0.8.
are you trying to do `(int)$item['subtotal'] === 0`?

Reply:
Yes since it will give 0.8, it means 0.8 is not equal to 0.00. So, we add the item to the product list we send to Atome. If we cast it to integer then it will treat non-zero (0.8) as zero.